zenoh/src/subscriber.rs

```diff
@@ -248,28 +164,11 @@ impl Drop for SubscriberInner<'_> {
     }
 }
 
-/// The mode for pull subscribers.
-#[non_exhaustive]
-#[derive(Debug, Clone, Copy)]
-pub struct PullMode;
-
-impl From<PullMode> for Mode {
-    fn from(_: PullMode) -> Self {
-        Mode::Pull
-    }
-}
-
 /// The mode for push subscribers.
 #[non_exhaustive]
 #[derive(Debug, Clone, Copy)]
 pub struct PushMode;
```

Comment:
This struct and the SubscriberBuilder parametrization should not be needed any more.

Reply:
Done